packages/bechamel-js/bechamel-js.0.3.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/mirage/bechamel"
+bug-reports:  "https://github.com/mirage/bechamel/issues"
+dev-repo:     "git+https://github.com/mirage/bechamel.git"
+doc:          "https://mirage.github.io/bechamel/"
+license:      "MIT"
+synopsis:     "HTML generator for bechamel's output"
+description: """A simple tool to generate a standalone HTML
+page which shows results from bechamel's benchmarks."""
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml"      {>= "4.08.0"}
+  "dune"       {>= "2.0.0"}
+  "result"     {>= "1.5"}
```

Comment:
I’m curious, why is result still needed?

Reply:
Hmmhmm, I'm not sure, I will check that in 1 hour.